### PR TITLE
Provide example for action summary and description

### LINKED ISSUE #1176: [3.0] Can't describe action in controller
there is no way to describe what endpoint actually do. in version 2 there was `description` property in `@ApiDoc` annotation
```
...
* @ApiDoc(
*  resource=true,
*  description="This is a description of your API method",
...
```

how can it be done now?

### PR DESCRIPTION
**Pull request by dbu**
fix #1176

@GuilhemN is this correct?

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -123,6 +123,10 @@ To document your routes, you can use the SwaggerPHP annotations and the
     class UserController
     {
         /*
+         * List the rewards of the specified user.
+         *
+         * This call takes into account all confirmed awards, but not pending or refused awards.
+         *
          * @Route("/api/{user}/rewards", methods={"GET"})
```
Comment (dbu) [inline Resources/doc/index.rst]: @GuilhemN does this actually work? the `user` parameter is not explained in the annotation.

Comment (GuilhemN) [reply Resources/doc/index.rst]: It is automatically added by the bundle to the documentation (as a string parameter).

Comment (dbu) [reply Resources/doc/index.rst]: okay. we should maybe say that, or even add an example for the in=path parameter annotation.

Comment (GuilhemN) [reply Resources/doc/index.rst]: Indeed, the docs are very minimal right now :)

**Comment by GuilhemN** [comment]
Thank you @dbu :)

**Pull request merged**